src/strings/en-us.json

```diff
@@ -553,6 +553,7 @@
     "LabelAudioCodec": "Audio codec",
     "LabelAudioLanguagePreference": "Preferred audio language",
     "LabelSelectAudioNormalization": "Audio Normalization",
+    "LabelSelectPreferredTranscodeAudioCodec": "Preferred transcode audio codec",
```

Comment:
Please be explicit that this only affects Audio track in Videos. Because the Audio transcoding will use a very different server-side logic.

Reply:
What if we add a header?
```html
    <div class="verticalSection verticalSection-extrabottompadding">
        <h2 class="sectionTitle">
            ${HeaderVideoAdvanced}
        </h2>

        <!-- select goes here -->

    <div>
```
Not sure about wording.

Reply:
If we are not rushing this with 10.9 release, then we can do a re-organize of the transcoding panel to make it easier to follow. It currently has a lot of options inside and a lot of options are actually very hard to find.

Reply:
Yes, ideally there should be a dedicated view to customize the device profile.

The problem with the 10.9 release is that we can't add translations to the release branch - Weblate can only push to the master branch, iirc.